inode/config.go

```diff
@@ -210,6 +220,28 @@ func Up(confMap conf.ConfMap) (err error) {
 			inodeCacheLRUItems:             0,
 		}
 
+		// TODO - should these config settings have defaults?
+		LRUCacheMaxBytes, err = confMap.FetchOptionValueUint64(volumeSectionName, "MaxBytesInodeCache")
+		if nil != err {
+			return
+		}
+		volume.inodeCacheLRUMaxBytes = LRUCacheMaxBytes
+
+		LRUDiscardTimeInterval, err = confMap.FetchOptionValueDuration(volumeSectionName, "InodeCacheEvictInterval")
+		if nil != err {
+			return
+		}
+		volume.inodeCacheLRUTickerInterval = LRUDiscardTimeInterval
+		volume.inodeCacheLRUTicker = time.NewTicker(volume.inodeCacheLRUTickerInterval)
+
+		go func() {
+			for range volume.inodeCacheLRUTicker.C {
+
+				// TODO - Do we have to worry about the volume being removed?
```

Comment:
Indeed - is there no sync.WaitGroup that the volume deletion code can at least Wait() on?

Reply:
Not that I am aware of...